docs/misc/declarative-configuration/config.yaml

```diff
@@ -0,0 +1,61 @@
+# This file serves as a specification how to map properties from the existing config properties
+# to the declarative configuration format.
+# This list does not contain all properties, but those that
+# are different from what the config bridge already does.
+
+# Based on https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/kitchen-sink.yaml
+
+# Re-use this flag to disable the Agent/Spring starter as well as the SDK.
+disabled: false # was: "otel.javaagent.enabled"
```

Comment:
I can't think of a good reason why you'd want separate flags for sdk and agent disable